pkg/state/broker.go

```diff
@@ -5,12 +5,149 @@
 
 package state
 
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/enmasseproject/enmasse/pkg/amqpcommand"
+	v1beta2 "github.com/enmasseproject/enmasse/pkg/apis/enmasse/v1beta2"
+
+	"pack.ag/amqp"
+)
+
+const brokerCommandAddress = "activemq.management"
+const brokerCommandResponseAddress = "broker_command_response"
```

Comment:
The reply address needs to be prefixed by activemq.management otherwise management responses will be dropped if the broker becomes full.